Remove -L<libdir> from default pcap-config --libs output, as libdir is
already in the default toolchain search path on Debian, and we want the
generated script to be arch-independent.

--- a/pcap-config.in
+++ b/pcap-config.in
@@ -11,7 +11,6 @@
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 includedir="@includedir@"
-libdir="@libdir@"
 LIBS="@LIBS@"
 LIBS_STATIC="@LIBS_STATIC@"
 VERSION="@PACKAGE_VERSION@"
@@ -64,20 +63,6 @@
 	shift
 done
 
-#
-# If we aren't installing in /usr, then provide a -L flag to let build
-# processes find our library.
-#
-# (We must check $prefix, as $libdir isn't necessarily /usr/lib in this
-# case - for example, Linux distributions for 64-bit platforms that
-# also provide support for binaries for a 32-bit version of the
-# platform may put the 64-bit libraries, the 32-bit libraries, or both
-# in directories other than /usr/lib.)
-#
-if [ "$prefix" != "/usr" ]
-then
-	LPATH=-L$libdir
-fi
 if [ "$static" = 1 ]
 then
 	#
@@ -90,16 +75,16 @@
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir $LPATH -l@PACKAGE_NAME@ $LIBS_STATIC"
+		echo "-I$includedir -l@PACKAGE_NAME@ $LIBS_STATIC"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir $LPATH $LIBS_STATIC"
+		echo "-I$includedir $LIBS_STATIC"
 	elif [ "$show_cflags" = 1 ]
 	then
 		echo "-I$includedir"
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "$LPATH -l@PACKAGE_NAME@ $LIBS_STATIC"
+		echo "-l@PACKAGE_NAME@ $LIBS_STATIC"
 	elif [ "$show_additional_libs" = 1 ]
 	then
 		echo "$LIBS_STATIC"
@@ -117,16 +102,16 @@
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir $LPATH -l@PACKAGE_NAME@ $LIBS"
+		echo "-I$includedir -l@PACKAGE_NAME@ $LIBS"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir $LPATH $LIBS"
+		echo "-I$includedir $LIBS"
 	elif [ "$show_cflags" = 1 ]
 	then
 		echo "-I$includedir"
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "$LPATH -l@PACKAGE_NAME@ $LIBS"
+		echo "-l@PACKAGE_NAME@ $LIBS"
 	elif [ "$show_additional_libs" = 1 ]
 	then
 		echo "$LIBS"
@@ -138,13 +123,9 @@
 	# cause all of its dynamic-library dependencies to be pulled in
 	# at run time.
 	#
-	# Do, however, include RPATH, to make sure that, on platforms
-	# that require this, programs built with this version of
-	# libpcap can find it at run time.
-	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir $LPATH @RPATH@ -l@PACKAGE_NAME@"
+		echo "-I$includedir -l@PACKAGE_NAME@"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
 		echo "-I$includedir"
@@ -153,6 +134,6 @@
 		echo "-I$includedir"
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "$LPATH @RPATH@ -l@PACKAGE_NAME@"
+		echo "-l@PACKAGE_NAME@"
 	fi
 fi
